specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx

```diff
@@ -135,9 +135,13 @@ export function FormTable<SCHEMA extends AnySchema>({
     addedResource.current = undefined;
     if (resourceIndex === -1 || rowsRef.current === null) return;
     const lastRow: HTMLElement | null = rowsRef.current.querySelector(
-      `:scope > :nth-child(${resourceIndex}) > [tabindex="-1"]`
+      `:scope > :nth-child(${resourceIndex + 1}) > [tabindex="-1"]`
     );
-    lastRow?.focus();
+    const firstFocusableElement = rowsRef.current.querySelector<HTMLElement>(
```

Comment:
why duplicate this complex code here?

please never copy complicated code - especially when it would be so easy to extract it into a function and reuse that cunction

Reply:
you need to look for first focusable in `lastRow`, not in `rowsRef.current` - that will solve your other problem

Reply:
it was to try the code for now I will extract it to a function once it works. Finding the first input or text area etc on lastRow doesn't work for a reason I didn't figure out yet.